### PR TITLE
Add kde-cli-tools to get keditfiletype5

### PR DESCRIPTION
**Pull request by travier**
Fixes: https://github.com/flathub/org.kde.dolphin/issues/57

### CODE REVIEW

**Comment by travier** [comment]
This needs plasma-workspace at build time only.

**Comment by Justinzobel** [comment]
bot, build

File: org.kde.dolphin.json [hunk 1]
```diff
@@ -141,6 +141,17 @@
                 }
             ]
         },
+        {
+            "name": "kde-cli-tools",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/plasma/5.23.3/kde-cli-tools-5.23.3.tar.xz",
```
Comment (Justinzobel) [inline org.kde.dolphin.json:150]:
```suggestion
                    "url": "https://download.kde.org/stable/plasma/5.27.6/kde-cli-tools-5.27.6.tar.xz",
```

File: org.kde.dolphin.json [hunk 2]
```diff
@@ -141,6 +141,17 @@
...
+                    "sha256": "4069a881b04f065bb56d5eae8ac2499fe276071e9350e23d919d4e62cd521c39"
```
Comment (Justinzobel) [inline org.kde.dolphin.json:151]:
```suggestion
                    "sha256": "b5e2b1c3bf82c112c8488aea73dca11a49963ea66e66db8f358f8e0394ba0faa"
```

**Review by Justinzobel — COMMENTED** [review]
Update kde-cli-tools to 5.27.6, this might resolve the issue.

**Comment by Justinzobel** [comment]
It needs plasma-workspace, so how do we get plasma-workspace in there at build-time only @travier?

**Comment by travier** [comment]
We need to add a "plasma-workspace" module above this change in the manifest and figure out which pieces of it are need for the `kde-cli-tools`. We probably don't want to have the full plasma-workspace installation in the Dolphin Flatpak.

**Comment by Justinzobel** [comment]
I might come back to this over the weekend to poke at it, see if I can't add it in.

**Comment by Justinzobel** [comment]
Replaced by #269.

**Pull request closed without merging**